### PR TITLE
Copyedit the README

### PR DESCRIPTION
**Pull request by dbgriff**
Added some punctuation and conjunctions. @samthor

### CODE REVIEW

**Comment by thagikura** [comment]
Thanks for the patch!

The change looks good, but can you please address the followings?

* Squash your commits into one commit
* Change the title of this PR to explain the PR better (something like "Copyedit the README")

File: README.md [hunk 1]
```diff
@@ -67,8 +67,7 @@ view.setLayoutParams(lp);
 #### Attributes for the FlexboxLayout:
 
 * flexDirection
-  * The direction children items are placed inside the Flexbox layout, it determines the
-  direction of the main axis (and the cross axis, perpendicular to the main axis).
+  * This attribute determines the direction of the main axis (and the cross axis, perpendicular to the main axis).The direction children items are placed inside the Flexbox layout.
```
Comment (dh--) [inline README.md]: Please add a space after the period on the first sentence.
").The"

**Comment by thagikura** [comment]
Friendly ping @dbgriff, are you able to address the comments?

**Comment by dbgriff** [comment]
Hi @thagikura, I changed the name of the PR and put all the edits under a new commit called Copyedit README, but I'm not sure how to get rid of the older commits I made previously.

**Comment by thagikura** [comment]
Ah ok. Looks like now GitHub supports squash commits from the Web UI. I'll do this when merging this.

Aside from that, this PR has a conflict with the latest master branch.
Could you resolve that? I think you can see the command instructions in this PR (The link as "Use the command line")

**Comment by dbgriff** [comment]
I can see the notification of the branch conflict that must be resolved but it says I need write access

**Comment by thagikura** [comment]
So is it possible to rebase your pull request on top of master and update this PR?

You can search bunch of web pages how to rebase a PR on top of master like this [one](http://stackoverflow.com/questions/17182624/contributing-to-project-on-github-how-to-rebase-my-pull-request-on-top-of-mast).

**Comment by dbgriff** [comment]
I've been reading up on it but I'm not very practiced in coding, I did all
my commits on the interface online. Is there any way to resolve the
conflicts just using the GitHub interface?

**Comment by thagikura** [comment]
As far as I know, the GitHub interface doesn't support resolving conflicts. But I highly recommend you to get used to using the usage of the git command line even if you don't code (Most of the documentations nowadays are managed though git).

Aside from that, let me confirm, did you sign the Contributor License Agreement  as mentioned in the googlebot above? 

I tried to merge this PR (https://github.com/google/flexbox-layout/pull/105), but  I couldn't confirm the CLA is signed.

**Comment by dh--** [comment]
@dbgriff when all else fails; scrap your change, and create a new pull request off of the latest revision. It's a bit of extra work, but considering the scope of your changes I think it will be faster than fighting with git.

Note: I did this with about half my changes when first learning git; I think I'm down to about 2% nowadays.

**Comment by dbgriff** [comment]
@thagikura, I completed my CLA. Will the #105 go through without problem now?

**Comment by thagikura** [comment]
Thanks. Merged #105. Closing this PR.

**Pull request closed without merging**